package/admin.py

```diff
@@ -0,0 +1,163 @@
+from django.contrib import admin, messages
+from django.db.models import Count, DecimalField, IntegerField, OuterRef, Subquery, Sum
+from django.http import HttpResponseRedirect
+from django.shortcuts import render
+
+from package.forms import BulkSendForm
+from package.models import Order, OrderItem
+
+
+class TotalPackagesFilter(admin.SimpleListFilter):
+    title = "total_packages_filter"
+    parameter_name = "total_packages_filter"
+
+    def lookups(self, request, model_admin):
+        return (
+            ("1", "1"),
+            ("2", "2"),
+            ("5", "5"),
+            ("10", "10"),
+            ("11", "10+"),
+        )
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value in ("1", "2", "5", "10"):
+            queryset = queryset.filter(_total_packages=value)
+        elif value in ("11",):
+            queryset = queryset.filter(_total_packages__gt=value)
```

Comment:
```suggestion
            queryset = queryset.filter(_total_packages__gte=value)
```

Reply:
otherwise we don't include case with 11 packages

Reply:
done + added test for this use case